### ISSUE TITLE
what is the command and what is the performance?

### ISSUE THREAD

**Issue by mit10000** [pyaf/DenseNet-MURA-PyTorch#1]
Thanks pyaf for sharing.
Have 3 questions:
1 what is the command to train and evaluate?
2 how good is the performance, in terms of F1 or accuracy...?
3 you train all images together, or only train on body part, such as train images of elbow, then train images of wrist,....?

Thank you for letting me know.

**Comment by pyaf** [comment]
Hi @mit10000 

1. I've added a `main.py` you can run to train the model. You can just run this `main.py` to train the model. Feel free to add a script for evaluation the model and raise a PR. 
2. Performance in terms of accuracy is about 80% (validation set). Rest I've added a Meter Class in train.py, you can see model performace in terms of precision, TP rates, FP rates etc.
3. I've trained the model only on Wrist dataset, as of now.

Let me know if you have any doubts.

**Comment by pyaf** [comment]
Hi @mit10000

1. I've added a main.py you can run to train the model. You can just run this main.py to train the model. Feel free to add a script for evaluation the model and raise a PR.
2. Performance in terms of accuracy is about 80% (validation set). You would have to calculate other metrics yourself. 
3 I've trained the model only on Wrist dataset, as of now.

Let me know if you have any doubts.

**Comment by mit10000** [comment]
Thanks for your help!

**Comment by pyaf** [comment]
@mit10000 do :star:  the repo, if you found it helpful :)

**Comment by mit10000** [comment]
Done!

**Comment by John1231983** [comment]
Hi. What is rank of your repo in the leaderboard? I wonder if your performance is close to the baseline one

**Comment by pyaf** [comment]
@John1231983 
It's 28

**Comment by John1231983** [comment]
@pyaf : Thanks. Is it 
```
Densenet121 (single model) Personal | 0.629
-- | --
```
Or
```
Bhaukali_v1.0 (single model) IIT BHU, Varanasi | 0.581
-- | --
```
Two results are very lower than the paper report.

**Comment by pyaf** [comment]
The second one.
Yeah that's true.

**Comment by John1231983** [comment]
Thanks. Could you suggest any reason why the reproduced result too low?
